panel/src/api/users.js

```diff
@@ -16,8 +16,11 @@ export default (api) => ({
 	async changeName(id, name) {
 		return api.patch("users/" + id + "/name", { name: name });
 	},
-	async changePassword(id, password) {
-		return api.patch("users/" + id + "/password", { password: password });
+	async changePassword(id, password, currentPassword) {
+		return api.patch("users/" + id + "/password", {
+			password: password,
+			currentPassword: currentPassword
```

Comment:
`Nitpick:` JS allows us to just reference the keys here (when key name and const/variable name are the same) and we usually do that when the whole object can be referenced by keys.
```suggestion
			password,
			currentPassword
```